test/slackbot.test.js

```diff
@@ -1,36 +1,123 @@
 /* global describe, it, beforeEach */
-var nock = require('nock');
 var Slackbot = require('../slackbot');
 var helpers = require('../helpers');
+var slack = require('@slack/client');
+var sinon = require('sinon');
 
-var slackApiOrigin = 'https://slack.com:443';
 var env = helpers.getEnv({
   env: {
     SLACK_TOKEN: 'test-token',
     NODE_ENV: 'development'
   }
 });
 
+function MockTransport (stubs) {
+  stubs = stubs || {};
+  this.RtmClient = function () {
+    this.on = sinon.spy(stubs.rtmOn);
+    this.start = sinon.spy(stubs.rtmStart);
+  };
+  this.WebClient = function () {
+    this.channels = {
+      info: sinon.spy(stubs.webChannelsInfo)
+    };
+    this.files = {
+      info: sinon.spy(stubs.webFilesInfo)
+    };
+  };
+}
```

Comment:
Hm, I'll look into it. I was thinking that you could only stub functions, but I forget that basically everything is a "function".